packages/launcher-client/src/client/impl/default.launcher.client.ts

```diff
@@ -80,15 +80,15 @@ export default class DefaultLauncherClient implements LauncherClient {
   }
 
   public async launch(payload: LaunchAppPayload): Promise<LaunchAppResult> {
-    let endpoint: string;
+    let endpoint = this.config.creatorUrl;
     let p: any = payload;
+
     if (payload.project.parts.length === 1 && payload.project.parts[0].shared.mission) {
-      endpoint = this.config.launcherURL;
+      endpoint = Locations.joinPath(this.config.launcherURL, '/launcher');
```

Comment:
is this needed? changing the `.env.staging` url should fix that no?

Reply:
This is needed we only need to add it here if we add it evey where the things like the get for the catalog will break